bin/ch/WScriptJsrt.cpp

```diff
@@ -725,7 +725,7 @@ bool WScriptJsrt::InstallObjectsOnObject(JsValueRef object, const char* name, Js
     memcpy(path, msg, (size_t)str_len);          \
     path[str_len] = char(0)
 
-void GetBinaryLocation(char *path, const uint32_t size)
+void GetBinaryLocation(char *path, const unsigned size)
```

Comment:
No worries @dilijev , luckily it is `unsigned` ;) Unsigned_MAX_Value + 1 --> 0 .. So we wouldn't use that buffer. Still, why we would ever have/need that big buffer for a file path / or how the system call itself can handle that nice piece of memory / is yet another PR discussion indeed :) Kudos to @digitalinfinity